README.md

```diff
@@ -53,6 +53,14 @@ npm run action src/www/storybook
 >
 > to your shell, then try `outline www/start`!
 
+## Passing configuration flags to actions
+
+Certain actions take configuration flags - but since we're running them through `npm`, you'll have to use the `--` seperator to funnel them through to the underlying process. For example, to set up a MacOS project in release mode, you'd run:
+
+```sh
+SENTRY_DSN=<your sentry dsn> npm run action cordova/setup macos -- --buildMode=release --versionName=<your version name>
```

Comment:
FYI, I added a command to the Apple readme that people can copy and past as-is: https://github.com/Jigsaw-Code/outline-client/blob/fortuna-readme/src/cordova/apple/README.md#set-up-xcode-project